spec/factories/user.rb

```diff
@@ -1,8 +1,17 @@
-# FactoryGirl.define do
-#   factory :user do
-#     name "John Doe"
-#     username  "JDoe"
-#     email "jdoe@example.com"
-#     password "password"
-#
-#   end
+FactoryGirl.define do
+  factory :default_user, class: User do
+    name      "John Doe"
+    username  "JDoe"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.